docs/enhancement-proposals/off-ep-0007.md

```diff
@@ -0,0 +1,116 @@
+no long-range vdW correction (`none`),
+an isotropic long-range vdW correction that integrates the vdW potential against the switching function out to infinite pair separation (`isotropic`),
+and the LJPME formulation (`LJPME`).
+
+## Usage and Impact
+
+While existing SMIRNOFF implementations always use a long-range isotropic dispersion correction (i.e. [OpenFF
+Toolkit](https://github.com/openforcefield/openff-toolkit/blob/0.10.6/openff/toolkit/typing/engines/smirnoff/parameters.py#L3695)),
+following discussion in [Standards #38](https://github.com/openforcefield/standards/pull/38), it was recognized that
+(1) there was no way to disable this long-range correction or specify an important alternative approach (LJPME) that leads to a different potential definition,
+(2) the exact form of the long-range isotropic dispersion correction currently used was not specified as part of the SMIRNOFF standard.
+Updating the specification would not impact this behavior if using the default, recommended setting.
+
+This proposed enhancement would provide a way to disable the long-range vdW correction as well as build force fields that use LJPME,
+which has been recognized as being important in treating heterogeneous systems such as [those containing lipid bilayers](https://doi.org/10.1021/ct400140n).
+
+Most engines implement a common form of isotropic long-range vdW correction in which the pairwise vdW potential is averaged over all vdW site pairs
+and integrated (numerically or analytically) alongside a surface area term (and a term to account for the switching function if one is in use) to account for the missing interactions out to infinite separation:
+```
+U_{correction} = [N (N-1)]^{-1} \sum_{i < j}^N \int_0^{\infty} dr [1-S(r)] [4 \pi r^2] U_{vdw}(r)
+```
+Here, `U_{correction}` is the long-range vdW correction, `N` is the number of vdW sites, `S(r)` is the switching function or cutoff function that assumes the value of `S(r) = 0` for `r \ge r_{cut}` and `S(r) = 1` for `r ~ 0`.
+
+In [OpenMM](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.NonbondedForce.html#openmm.openmm.NonbondedForce.setUseDispersionCorrection),
+this is controlled via a method that accepts a boolean toggle. Similar options are implemented
+in [GROMACS](https://manual.gromacs.org/current/user-guide/mdp-options.html#mdp-DispCorr), AMBER
+(`nodisper`), [LAMMPS](https://docs.lammps.org/pair_modify.html), and likely other engines. However
+unlikely, if in the future other types of corrections than the current isotropic one are developed,
+this specification can be extended because the setting is a string value, not a boolean.
+
+Many of these engines also provided the capability to use LJPME.
+As with electrostatics, we do not specify the exact parameters used for PME (real-space cutoff, grid spacing, error tolerance);
+the SMIRNOFF spec instead specifies the Ewald sum is the true desired potential and the simulation engine must make a faithful
```

Comment:
There is also particle-particle particle mesh (P3M) as employed by LAMMPS as the default long range, which should converge to the PME result in the limit of fine-grain parameters, it should probably mention "or equivalent to PME" or something along those lines.